.github/workflows/master.yml

```diff
@@ -176,6 +176,17 @@ jobs:
           echo $env:computerName
           ls 
           $ErrorActionPreference = "SilentlyContinue"
+          cd minikube_binaries
+          ls
+          $env:KUBECONFIG="${pwd}\testhome\kubeconfig"
+          $env:MINIKUBE_HOME="${pwd}\testhome"
+          .\minikube-windows-amd64.exe delete --all --purge
+          Get-VM | Where-Object {$_.Name -ne "DockerDesktopVM"} | Foreach {
+            minikube-windows-amd64.exe delete -p $_.Name
+            Suspend-VM $_.Name
+            Stop-VM $_.Name -Force
```

Comment:
For clarity, I think this should specify `-Name`, as per https://docs.microsoft.com/en-us/powershell/module/hyper-v/stop-vm?view=win10-ps

I think `-TurnOff` may be a valuable flag as well.